GomokuClient/packages/gomoku-core/src/shared/lib/utils.ts

```diff
@@ -39,13 +39,7 @@ export interface FixedKeys {
   chunkReloadedAt: string;
 }
 
-export interface DynamicKeys {
-  [key: `gameBoard_${string}`]: string;
-  [key: `nextTurn_${string}`]: string;
-  [key: `lastTile_${string}`]: string;
-}
-
-export type Schema = FixedKeys & DynamicKeys;
+export type Schema = FixedKeys;
```

Comment:
Right now there is no point to have FixedKeys as separate interface, let's put it directly in to the type

```ts
export interface Schema  {
  jwtToken: string;
  currentGameID: string;
  chunkReloadedAt: string;
}
```